arangodb-net-standard/GraphApi/GraphApiClient.cs

```diff
@@ -405,5 +405,38 @@ public async Task<GetGraphEdgeCollectionsResponse> GetEdgeCollectionsAsync(strin
                 throw await GetApiErrorException(response);
             }
         }
+
+        /// <summary>
+        /// Removes a vertex from the collection.
+        /// DELETE/_api/gharial/{graph}/vertex/{collection}/{vertex}
+        /// </summary>
+        /// <param name="graphName"></param>
+        /// <param name="collectionName"></param>
+        /// <param name="vertexKey"></param>
+        /// <param name="query"></param>
+        /// <returns></returns>
+        public async Task<DeleteVertexResponse<T>> DeleteVertexAsync<T>(
```

Comment:
Should we provide an overload with document-handle? I've seen that in one of your other PR and thought it was a good idea.

Reply:
Created #146